moped-editor/src/queries/project.js

```diff
@@ -51,10 +49,11 @@ export const SUMMARY_QUERY = gql`
       contractor
       purchase_order_number
       work_assignment_id
-      moped_proj_features(where: { status_id: { _eq: 1 } }) {
-        feature_id
-        project_id
-        location
+      moped_proj_components(where: { status_id: { _eq: 1 } }) {
+        moped_proj_features(where: { status_id: { _eq: 1 } }) {
+          feature_id
+          feature
+        }
```

Comment:
as i mention in the commit for this change—querying `moped_proj_features` through `moped_proj_components` helps keep the shape of data we work with consistent, and is probably limiting some future tech debt, since the direct relationship b/t `moped_proj_features` and `moped_project` is superfluous and imo should be deprecated. every `moped_proj_feature` must have a `moped_proj_component` that in turn must have a `moped_project`.